packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -43,8 +43,8 @@ Okta Aerial introduces a few terms to the Okta ecosystem:
 
 | Term                                                           | Definition                                                                    |
 |----------------------------------------------------------------|--------------------------------------------------------------------------------|
-| Aerial account              | The management layer around multiple orgs within Okta. The Aerial account lives outside of your orgs and can manage any production or preview org linked to the Aerial account. |
-| Aerial org        | Holds the [authorization server](/docs/concepts/auth-servers/) for all Aerial API actions in any org in the Aerial account. Choose one org to permanently serve as the Aerial Org. <br/>[Super admins](https://help.okta.com/okta_help.htm?type=oie&id=ext_superadmin) can create API clients in the Aerial Org to access the Aerial account. The Aerial Org also contains all [System Log](https://help.okta.com/okta_help.htm?type=oie&id=ext_Reports_SysLog) events associated with Okta Aerial actions. |
+| Aerial account              | The management layer around multiple orgs within Okta. The Aerial account lives outside of your orgs and can manage any production or preview org linked to the Aerial account. Each Aerial account has a dedicated Aerial org.  |
+| Aerial org        | The Aerial org is used to register API clients and acts as an [authorization server](/docs/concepts/auth-servers/) for all API calls made in the associated Aerial account. The Aerial org also contains all [System Log](https://help.okta.com/okta_help.htm?type=oie&id=ext_Reports_SysLog) events associated with Okta Aerial actions. |
```

Comment:
I remember working with Barbara on these and she reminded me that it's a bit redundant to begin the definition by restating the term.
```suggestion
| Aerial org        | Used to register API clients and acts as an [authorization server](/docs/concepts/auth-servers/) for all API calls made in the associated Aerial account. The Aerial org also contains all [System Log](https://help.okta.com/okta_help.htm?type=oie&id=ext_Reports_SysLog) events associated with Okta Aerial actions. |
```